dallinger/recruiters.py

```diff
@@ -604,6 +614,24 @@ def get_qualifications(self, hit_id, sandbox):
         }
 
 
+class DevProlificRecruiter(ProlificRecruiter):
+    """A debug recruiter for [Prolific](https://app.prolific.com/)"""
+
```

Comment:
When setting

```
recruiter = generic
debug_recruiter = devprolific
```

I see log messages like this:

```
INFO:/Users/peter/Dallinger/dallinger/recruiters.py:Initializing GenericRecruiter...
INFO:/Users/peter/Dallinger/dallinger/recruiters.py:Initializing DevProlificRecruiter...
```

which suggests that `GenericRecruiter` is still being used for some operations. Maybe it is worth double-checking (e.g. by putting a breakpoint inside `GenericRecruiter.__init__`) to make sure nothing fishy is happening here?

Reply:
I implemented your suggestion above with just setting
`recruiter = prolific`
and the output now is
```
INFO:/home/frank/projects/Dallinger/dallinger/recruiters.py:Initializing ProlificRecruiter...
INFO:/home/frank/projects/Dallinger/dallinger/recruiters.py:Initializing DevProlificRecruiter...
```